### PR TITLE
jsx description

### PR DESCRIPTION
**Pull request by AkifumiSato**
I wanted an explanation of the `jsx` function, so I added it.

### CODE REVIEW

**Comment by satyarohith** [comment]
Thanks for your contribution. Just a few comments.

File: readme.md [hunk 1]
```diff
@@ -22,3 +22,25 @@ serve({
...
+render with `preact` to create html Response
+
+```js
+import {
+  jsx,
+  serve,
+} from "https://deno.land/x/sift@0.1.7/mod.ts";
+import { html } from "https://cdn.skypack.dev/htm@v3.0.4/preact/standalone.module.js";
+
+const App = html`
```
Comment (satyarohith) [inline readme.md:37]: Deno Deploy supports JSX/TSX out of the box. The file extensions need to be either `.jsx/tsx` for Deno Deploy to process the code. So we don't need to use the `htm` package.

File: readme.md [hunk 2]
```diff
@@ -22,3 +22,25 @@ serve({
   404: () => new Response("not found"),
 });
 ```
+
+### `jsx()`
+
+render with `preact` to create html Response
```
Comment (satyarohith) [inline readme.md:28]:
```suggestion
Render JSX components to produce Response with `text/html` content type.
```

The user can also use `react` so JSX components might be more appropriate here.

File: readme.md (hunk 1 above)
Comment (AkifumiSato) [reply readme.md]: Is my fix okay?

File: readme.md [hunk 3]
```diff
@@ -22,3 +22,22 @@ serve({
   404: () => new Response("not found"),
 });
 ```
+
+### `jsx()`
+
+Render JSX components to produce Response with `text/html` content type.
+
+```js
+import { jsx, serve } from "https://deno.land/x/sift@0.1.7/mod.ts";
```
Comment (satyarohith) [inline readme.md:31]:
```suggestion
import { jsx, serve, h } from "https://deno.land/x/sift@0.1.7/mod.ts";
```

Sorry, I didn't mention it before but we can import `h` from sift. Sift uses a modified version of preact with types that are supported on Deno.

File: readme.md [hunk 4]
```diff
@@ -22,3 +22,22 @@ serve({
...
+import { h } from "https://cdn.skypack.dev/preact@10.5.13";
```
Comment (satyarohith) [inline readme.md:32]:
```suggestion
```

**Review by satyarohith — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**